_config.yml

```diff
@@ -61,3 +61,5 @@ exclude:
   - repos/*.html
   - repos/*.md
 
+repository:
+  istio/istio.github.io
```

Comment:
What's this for?

Reply:
this is to avoid some NGO liquid error @kyessenov was getting because his git origin wasn't set

Reply:
breaks on my machine without it. although i don't know if it's ok to check-in.